lyse/dataframe_utilities.py

```diff
@@ -18,6 +18,31 @@
 from labscript_utils.properties import get_attributes
 from labscript_utils.shot_utils import get_shot_globals
 
+def rangeindex_to_multiindex(df, inplace):
+    if isinstance(df.index, pandas.MultiIndex):
+        # The dataframe has already been converted.
+        return df
+    try:
+        padding = ('',)*(df.columns.nlevels - 1)
+        try:
+            integer_indexing = _labconfig.getboolean('lyse', 'integer_indexing')
+        except (LabConfig.NoOptionError, LabConfig.NoSectionError):
+            integer_indexing = False
```

Comment:
Just had a go at testing this in lab and had a failure here. Neither `_labconfig` nor `LabConfig` are defined in `dataframe_utilities.py`

Reply:
Thanks for finding this.  I don't understand what is going on here that these errors are not appearing when I am doing local testing (nor why my editor is not seeing them).

Reply:
This is still only partially fixed. `_labconfig` is not defined in `dataframe_utilities.py` either. 

Not sure why things aren't tripping for you. This particular error pops for me when running a multi-shot analysis after importing a large batch of runs (which had multiple single-shot analyses run on them). I suppose there is also a chance that we have slightly different labconfig values for lyse that are leading to slightly different behavior.

Reply:
This fix was a bit less trivial (see commit log).  To avoid circular imports I moved `_labconfig` and `_lyse_port` to utils.py and changed them to `LABCONFIG` and `LYSE_PORT` to mark them as globals.  For the same reason I moved `LYSE_PATH` to the same place.  For testing I ran on both a PC and a MAC and did both single and multishot scripts, so hopefully there are no more problems.